configs/components/openssl-3.0.rb

```diff
@@ -133,13 +133,26 @@
     '--libdir=lib',
     "--openssldir=#{settings[:prefix]}/ssl",
     'shared',
-    'no-asm',
+    'no-gost',
     target,
     sslflags,
     'no-camellia',
     'no-ec2m',
     'no-md2',
-    'no-ssl3'
+    'no-ssl3',
+    'no-ssl3-method',
+    'no-dtls1-method',
```

Comment:
```suggestion
    'no-dtls',
    'no-dtls1-method',
```

Reply:
`no-dtls` was already specified [here](https://github.com/tvpartytonight/puppet-runtime/blob/PA-5409/configs/projects/_shared-agent-settings.rb#L229), so I wasn't sure if we wanted to repeat them?

Reply:
I think it is the same for the other suggestions you have; they are already in the `shared-agent-settings.rb`.

Reply:
ohhh, right I forgot about `openssl_extra_configure_flags`